types/config.go

```diff
@@ -206,6 +209,12 @@ type RunConfig struct {
 	// InstanceGroup
 	InstanceGroup string
 
+	// AwsIPN is Aws instance profile name
+	AwsIPN string
+
+	// AwsASG is Aws auto scaling group
+	AwsASG string
```

Comment:
I think InstanceGroup is the equivalent being used on GCP so we should use that instead